### ISSUE TITLE
Checked✅ item saved as a not checked item in local storage

### ISSUE THREAD

**Issue by amjadnoor** [yashsehgal/todo_application#4]
## Description
When an item is checked they don't saved as a checked item in Local Storage. When you re-visit the website the item will sown but also checked item is sown as not checked.

**Comment by yashsehgal** [comment]
> ## Description
> When an item is checked they don't saved as a checked item in Local Storage. When you re-visit the website the item will sown but also checked item is sown as not checked.

Yeah, I was also thinking to implement this feature. You can get started with this one, It would be very helpful 👍🏽
